x-pack/plugins/ml/public/embeddables/anomaly_explorer/anomaly_explorer_initializer.tsx

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiForm,
+  EuiFormRow,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiFieldNumber,
+  EuiFieldText,
+  EuiModal,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { AnomalyExplorerEmbeddableInput } from '..';
+import { DEFAULT_MAX_SERIES_TO_PLOT } from '../../application/services/anomaly_explorer_charts_service';
+
+const MAX_SERIES_ALLOWED = 48;
+export interface AnomalyExplorerInitializerProps {
+  defaultTitle: string;
+  initialInput?: Partial<Pick<AnomalyExplorerEmbeddableInput, 'jobIds' | 'maxSeriesToPlot'>>;
+  onCreate: (props: { panelTitle: string; maxSeriesToPlot?: number }) => void;
+  onCancel: () => void;
+}
+
+export const AnomalyExplorerInitializer: FC<AnomalyExplorerInitializerProps> = ({
```

Comment:
As before, I think `AnomalyChartsInitializer` might be a better name for this if it is only containing the charts from the Anomaly Explorer.

Reply:
Renamed here [`3c31e37` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/3c31e37dacc795ad9e4b0b71d459616bda58b979)